packages/core/services/FileService/DatabaseFileService/index.ts

```diff
@@ -110,6 +111,7 @@ export default class DatabaseFileService implements FileService {
      * and potentially starting from a particular file_id and limited to a set number of files.
      */
     public async getFiles(request: GetFilesRequest): Promise<FileDetail[]> {
+        if (!this.dataSourceNames.length) return [];
```

Comment:
The changes in this file are actually totally unrelated so I can definitely separate it into its own ticket, but I've found that I frequently can't load the app at all (fails on render) unless these two lines are there. Is this just on mine/is there another way around this?

Reply:
Separating this into its own ticket bc relates to the issue Tyler is having on Mac